src/pages/Checkout/CheckoutPage.vue

```diff
@@ -792,7 +793,9 @@ export default {
 				query: upsellQuery,
 				fetchPolicy: 'network-only',
 			}).then(({ data }) => {
-				this.upsellLoan = data?.lend?.loans?.values[0];
+				const loans = data?.lend?.loans?.values;
+				// Temp solution so we don't show reserved loans on upsell
+				this.upsellLoan = loans.filter(loan => isLoanFundraising(loan))[0] || data?.lend?.loans?.values[0];
```

Comment:
I would think the OR condition here should actually be an empty object or something that can turn off the visual result. If the first check doesn't pass that means you'd be showing a loan with some funny state...

Reply:
ya good point - also added another conditional to the `v-if` statement to check for this for the upsell module